qa/resources/template.conf

```diff
@@ -37,4 +37,23 @@ scorex {
+                      "00000060ac3ad48d23802cb4d73c40f55c067f29c4eae5530d75e25a2754f6cc7e217c79af91a68a6aeff79e3ee96435eb564e1225a932a7cea7514f6b3325c51b97088287ce134405d55d033d76785e82668fe929176f3e74f0711cdc1e8376e53b0100fe903094bd158a3432cabfdd1f95d2380bf64029992be645d1a1dbdfa5c773e50351d6231572eb1278685edb3e871f1c2d2928d4438d88b92e7e86581821ed42a3397dc4b36b2a02132b0c9f039fedbd68bf0de450d4902cea5b28a2a99e0000965a80d923cc25795352725332e933e23152a663f09c2851cdd31858389d3cf7610916a8d920951c91393522a5c37add5ff533c4e89aa7e4a68629bfc977ed732e93c4a21c04a7d6f2ec297aacd6f42a37aeb765f25b5836957128bf7496000000",
+                      "000000603bfc9513a3d5b9ab0725dfc4162dd90dfb347fe5a1acce627e86d4b0795a6270f749b512b43bb68532e1bcbc5c025ae5255e6a305444a1282680da070eb94b1ead199fc2456f75a097a653f4ce8a4f6dc0c36685664ad9a3216efc8fda8601005e774bf505ed6d88d650df9d94e806745ef73b548936c89a7ccd7a04cc4dcd8e9f526de4d775023ae841203554e2992124766e244be94982c9419843aef03e7515493d3e4ea7109a7956eabaeb672d237a94fe71d36f7d2d625116a9a2dd0000c4483303bc9b5a90547d1bf5f9045c1f1b4a731835e229cbca4f92924ac9a7c62349e8ebca65937d3b795900d5f9f57fc984d4c17d1e1fd61626c8920ff2cfd63a2f4c9cf51f4776abbf4f2ba1250da11f4f5dd0f1ce4b0f40e8697405ad010000",
+                      "00000060b4c74b89820da2faf5bcacca7a521811bdb4eb50ecbc95575507fedd66dbe24ee2f278c14adc9f695568bcb9298c21feb4d4302770fc0b2a3da666f21c751b01953a6a6c03d8ad428009d5227bb074c2b205c8893363febc69df57f0d79801001816aa377751ebc6a3ca6222a397581964b12236be2020a67f6ce1a7fccf5ae576682ef1d9278b844c8c5323c948f47d3ad3e12cc3db5fe26adb68a1dd867296c4d5f6b5259db0e5c0814f02ce6f62a5efd76f62b790990a41a00477c7160000eee8e6755da744883ad946d5376dec55052e000ec3f5bdf18085abcb76851f1aa73fff91d45f58bf43cfc2a4db86c2bc5afe243a0d5aba1f658697abde21a88315711a8fe138bd3ed76601fba97a426dff1b20e3bd0e1de4341ee081225a010000",
+                      "00000060cc4fd452e68d651b8cb9d4457231b31fd5894dee227c90b5bdb8ab6448a8476227aa549d078137981f9b6bb7d9e08a86de941b8d02a93416238cabd39f559fbb0089b3c8a564f695bf7160f9592857b95f775dcdf72fb974be1ae8cd989c01007d35e074db99177c1aa34d8c809051a65dfff8abbde0467c97c0dd3088e6c91e44bc22281c159cf00fc96a9c85cc604b05c422dab44d5ba90b4c94713ab5a229c97017f00a494e75d32e4158fe9c8de5893041ad583237b3fa71bb679e990100d22d2da856042566c963a46593e3c05822533875dd02de4a431b989a9092fed1c3e4c7e902f29b49965cc7772f3217a8fecd394ccae9d35127ae344c0dc19862e7e123e54c00d579eb98b3621e74c2cf4a0e048e230cc578022de04170ba000000"
+                     ]
+    provingKeyFilePath = ""
+    verificationKeyFilePath = ""
```

Comment:
look like without correct proving/verification keys that test shall be failed everytime.